### DISCUSSION TITLE
Please create this app for windows too

### DISCUSSION THREAD

**Discussion by Uzcoin404** [focustimerhq/FocusTimer#671] [Ideas]
I really like this app but it's for only linux OS I couldn't find any programs like this in Windows

**Comment by kamilprusko** [comment]
Such app relies heavily on system integration. I read that core bits are doable with the windows API:

- indicator - https://learn.microsoft.com/en-us/windows/win32/shell/notification-area
- notifications - https://learn.microsoft.com/en-us/windows/win32/shell/quickstart-sending-desktop-toast
- detecting idle time - https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getlastinputinfo
- detecting system suspend - https://learn.microsoft.com/en-us/windows/win32/power/system-power-states

But, in my opinion its best to keep the app native. For one I don't know how to mix vala with the windows API; and second - I don't have any windows machine to try it. So, I have to dismiss this request.

**Reply by Uzcoin404** [reply]
I'm trying to use windows subsystem for linux to run app. it would be great if the app has a windows version even if won't work as the same as native app
